relay-kafka/src/producer/mod.rs

```diff
@@ -402,7 +402,12 @@ impl Producer {
                 relay_log::error!(
                     error = &error as &dyn std::error::Error,
                     tags.variant = variant,
-                    "error sending kafka message"
+                    tags.topic = topic_name,
```

Comment:
FWIW, the error was already tagged by `variant`, which describes what data type was dropped (e.g. `"metric"`).